ui_framework/src/components/table/listing_table/listing_table_tool_bar_footer.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiToolBarFooter,
+  KuiToolBarText,
+  KuiToolBarFooterSection,
+} from '../../';
+
+export function KuiListingTableToolBarFooter({ pager, itemsSelectedCount }) {
+  return (
+    <KuiToolBarFooter>
+      <KuiToolBarFooterSection>
+        <KuiToolBarText>
+          {itemsSelectedCount === 1 && '1 item selected'}
+          {itemsSelectedCount > 1 && `${itemsSelectedCount} items selected`}
```

Comment:
This seems a little odd because these two conditions are implicitly mutually exclusive, but the second one will still be evaluated even if the first one is true. How about making it more explicit with a function?

```jsx
export function KuiListingTableToolBarFooter({ pager, itemsSelectedCount }) {
  const renderText = () => {
    if (itemsSelectedCount === 1) {
      return "1 item selected";
    }

    return `${itemsSelectedCount} items selected`;
  };

  return (
    <KuiToolBarFooter>
      <KuiToolBarFooterSection>
        <KuiToolBarText>
          {renderText()}
        </KuiToolBarText>
      </KuiToolBarFooterSection>

      <KuiToolBarFooterSection>
        {pager}
      </KuiToolBarFooterSection>
    </KuiToolBarFooter>
  );
}
```

Reply:
Yep, getting too fancy! :)  It's more readable your way, will update.